documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -3228,7 +3228,12 @@ If the connector cannot find the publication, the connector throws an exception
 If no publication exists, the connector creates a new publication for tables that match the current filter configuration as specified by the `schema.include.list`, `schema.exclude.list`, and `table.include.list`, and `table.exclude.list` connector configuration properties.
 For example: `CREATE PUBLICATION <publication_name> FOR TABLE <tbl1, tbl2, tbl3>`.
 If the publication exists, the connector updates the publication for tables that match the current filter configuration.
-For example: `ALTER PUBLICATION <publication_name> SET TABLE <tbl1, tbl2, tbl3>`.
+For example: `ALTER PUBLICATION <publication_name> SET TABLE <tbl1, tbl2, tbl3>`. +
+ +
+`no_tables` - If a publication exists, the connector uses it.
+If a publication does not exist, the connector creates a publication without specifying any table.
+For example: `CREATE PUBLICATION <publication_name>;`.
+This can be useful if the connector should only capture logical decoding messages and is not interested of events in any table. In this case, you may also need to specify, for example, `"table.exclude.list": "public.*"` or `"schema.exclude.list": "public"`; that allows the connector not to emit and process `READ` events from all tables.
```

Comment:
> to make a distinction between capturing events and capturing logical decoding messages

Actually, logical decoding messages are a type of events; I will clarify that in the text.

> When we set the no_tables option, if there's no existing publication, the connector will create a publication that does not specify which tables to include in it ... the resulting publication would then include ... all tables? 

Using this new Debezium Postgres connector config option, a publication will be created with literally no tables. As said in the https://www.postgresql.org/docs/current/sql-createpublication.html[docs]:

> If FOR TABLE, FOR ALL TABLES or FOR TABLES IN SCHEMA are not specified, then the publication starts out with an empty set of tables.

This primarily addresses a very specific use case where database/WAL changes should be triggered not by storing rows in Postgres tables, but by using the Postgres `pg_logical_emit_message` function, which allows writing directly to WAL. Using this option causes the connector to not emit any `CREATE`, `UPDATE`, or `DELETE` events on any table.

But `READ` events based on reading rows in tables are published regardless of the publication creation options. If we don't specify `schema/table` filters, it is still possible for the connector to emit and process `READ` events from tables.

So if the user is not interested in any "write" (that is, publication-related) events, they may not be interested in any read events either, so I've additionally specified what they need to do to prevent `READ` events from being processed.